docs/streamen.md

```diff
@@ -16,3 +16,15 @@ Dit document is opgesteld om wat duidelijkheid te bieden wat betreft het streame
 ## Sancties
 
 * Bij het streamen van een drugs locatie zal er een ban van een week uitgedeeld voor de eerste keer bij een volgende overtreding zal er overworden gegaan naar een ban van een maand.
+
+## Bepaling en rechten
+
+1. Opgenomen of gepubliceerd materiaal zoals video, audio of elk ander beeldmateriaal gecreëerd door het gebruiken van onze diensten, blijft voor de duur van 12 maanden intellectueel eigendom van TedeaPolis.
+1. TedeaPolis behoud zich het recht tot het eisen van verwijdering binnen 24 uur van video, audio of beeldmateriaal in welke vorm dan ook bij overtreding van de Algemene Plaatselijke Verordening TedeaPolis 2020 te vinden op https://wetboek.tedeapolis.nl/apv/ . TedeaPolis geeft de gebruiker dus 24 uur om de publicatie offline te halen.
```

Comment:
kan de nummering gewoon 12345 ipv 11111

Reply:
Is officiele markdown nummering: https://guides.github.com/features/mastering-markdown/
Dus in preview staat t gewoon goed